containers/podman-desktop
# feat: add compatibility mode enable / disable button for macOS

feat: add compatibility mode enable / disable button for macOS

### What does this PR do?

* Adds a compatibility mode enable / disable button (with modal) for
  macOS users
* Runs podman-mac-helper binary that's presumingly installed via podman
  brew or pkg
* Errors out / outputs on success

In the future we will implement support for Windows and Linux
compatibility enablement.

### Screenshot/screencast of this PR



https://user-images.githubusercontent.com/6422176/225044544-63ff628b-e878-4a87-b6cb-5f15dd388e05.mov



### What issues does this PR fix or reference?

Fixes part of: https://github.com/containers/podman-desktop/issues/1447



### How to test this PR?

Press the button on macOS via `yarn watch`. You should be able to
disable / enable compatibility mode successfully by following the prompt
instructions.
